superset-frontend/spec/javascripts/sqllab/actions/sqlLab_spec.js

```diff
@@ -107,15 +107,6 @@ describe('async actions', () => {
       });
     });
 
-    it.skip('parses large number result without losing precision', () =>
```

Comment:
I'd rather leave this skipped than delete it, in hopes that we can rekindle it.

Reply:
Should we disable this rule then?

Reply:
I thought that this was old code because we don't have many skipped tests. I'll restore skipped tests and try to make them work.

Reply:
I'm OK with making them work in separate (individual) PRs, so we can get this PR pushed through.

Reply:
I'll restore the tests and keep the warnings. Later we can create a specific PR to fix these skipped tests.